src/main.js

```diff
@@ -80,8 +104,62 @@ const findDoubleWinner = ([p1Name, p1a, p1b], [p2Name, p2a, p2b]) => {
 
 }
 
+// check that players own the correct tokens
+const checkTokenType = (tokenList, tokenType, playerLedger, playerName) => {
+  let allTokensOwned = true
+  tokenList.map( t => {
```

Comment:
I would probably use `forEach` here instead of `map`. Map returns a list of the results generated by each function call. `forEach` simply calls the function on each element without gathering up the results.

Reply:
Done.